contributing/DEVELOPMENT.md

```diff
@@ -56,7 +50,7 @@ $ make testacc
 
 [Development overrides for provider developers](https://www.terraform.io/docs/cli/config/config-file.html#development-overrides-for-provider-developers) can be leveraged in order to use the provider built from source.
 
-To do this, populate a Terraform CLI configuration file (`~/.terraformrc` for all platforms other than Windows; `terraform.rc` in the `%APPDATA%` directory when using Windows) with at least the following options:
+To do this, populate a Terraform CLI configuration file (`~/.terraformrc` for all platforms other than Windows; `terraform.rc` in the `%APPDATA%` directory when using Windows) with at least the below options and also set `TF_CLI_CONFIG_FILE=$HOME/.terraformrc`.
```

Comment:
Please add a link to the documentation: https://www.terraform.io/docs/cli/config/config-file.html
Also, setting `TF_CLI_CONFIG_FILE` shouldn't be necessary if the configuration file is correctly located for your platform (I just verified this on mac).

Reply:
this seems to be a bug: https://github.com/hashicorp/terraform/issues/30169

I will update the docs to remove that env var